cached-mamba/action.yml

```
@@ -31,13 +31,15 @@ runs:
       channel-priority: strict
       activate-environment: my-env
       use-mamba: true
+  - name: Set cache date and number
```

```suggestion
  - name: Set cache date
```

I guess it's just the date

---

Ah yes, you removed the number at some point.